pages/iam/concepts.mdx

```diff
@@ -54,27 +54,29 @@ You are the [Owner](#owner) of the Organization that is created with your Scalew
 
 Similarly, you can invite other users to be Guests in your Organization. Whereas Owners have full rights and access to all resources and features in their Organization, Guests have only the rights and permissions given to them via [policies](#policy).
 
-<Lightbox src="scaleway-iam-owners-guests.webp" alt="" />
+<Message type="important">
+  Guests will be deprecated by July 2025. From then on, only Members can be created as users of your Organizations.
+</Message>
 
 ## IAM
 
 **I**dentity and **A**ccess **M**anagement allows you to share access to the management of your Scaleway [resources](#resource) in a controlled and secure manner.
 
-This is achieved by inviting users to be Guests in your account's [Organization](#organization), and creating [policies](#policy) that define in a very fine-grained way exactly what permissions they should have for which [resources](#resource) in which of your [Projects](#project) or across your whole Organization.
+This is achieved by adding members to your [Organization](#organization), and creating [policies](#policy) that define in a very fine-grained way exactly what permissions they should have for which [resources](#resource) in which of your [Projects](#project) or across your whole Organization.
 
-Similarly, you may participate as a Guest in someone else's Organization, where you will have the precise rights that they accord to you using policies.
+Similarly, you may be added as a Member in someone else's Organization, where you will have the precise rights that they accord to you using policies.
 
 You can also create non-human users in your Organization, called [IAM applications](#application), in order to give applications programmatic access to your Scaleway resources.
 
 ## Member
 
-You are a Member when you are added to an Organization by an Owner or user with IAM Manager permissions. Members exist only within the specific Organizations in which they are created. This is one of the methods employed at Scaleway to allow Organizations to have multi-users. Members fulfill the same purpose as Guest, while ensuring the security of the Organization.
+You are a Member when you are added to an Organization by an Owner or user with IAM Manager permissions. Members exist only within the specific Organizations in which they are created. This is one of the methods employed at Scaleway to allow Organizations to have multi-users.
```

Comment:
This is one of the methods employed at Scaleway to allow Organizations to have multiple users.
-> this is the only way so maybe we can remove this sentence